### ISSUE TITLE
Possible license conflict

### ISSUE THREAD

**Issue by moonlitpath** [shimmerproject/elementary-xfce#493]
I just tried to upgrade my Gentoo system and it wouldn't let me upgrade this icon pack because it is masked by non-free CC-BY-NC-SA license. I thought these icons were GPL-licensed? I am not a lawyer, but as far as I know GPL is not compatible with CC-BY-NC-SA.

**Comment by kuraga** [comment]
Fixed by #496?

**Comment by newhoa** [comment]
> Fixed by [#496](https://github.com/shimmerproject/elementary-xfce/pull/496)?

Yep! Should be fixed. The newest release, 0.20.1, should resolve this. Thanks for the report. Let us know if there are any more concerns of this sort. Thanks!
